JavaScript and Ajax/DeleteStudentRecord.page

```diff
@@ -0,0 +1,45 @@
+<apex:page >
```

Comment:
Some points from assignments are missing 

- why are we opening in new window?
- No records selected error message is missing.
- Cancel button does not work due to obvious reason (check comment on cancel method)